.github/workflows/packer.yml

```diff
@@ -38,18 +38,23 @@ jobs:
         --passwd-on-stdin --background \
         --servercert ${{ secrets.ORKA_VPN_SERVER_CERT }}
 
-    - name: Init Packer
-      run: packer init orka.pkr.hcl
+    - name: Init Packer base
+      if: matrix.os != 'sonoma-arm64' || 'sequoia-arm64'
+      run: packer init orka-base.pkr.hcl
       working-directory: ansible/packer
     
     - name: Run Packer (base image)
       # Skip arm64 for now as Xcode download takes too long
-      if: matrix.os != 'sonoma-arm64'
+      if: matrix.os != 'sonoma-arm64' || 'sequoia-arm64'
```

Comment:
Just want to confirm - is this syntax definitely correct for checking that it's not either of those two strings? It's quite unusual.

Reply:
Ive changed it to `if: ${{ matrix.os != 'sonoma-arm64' && matrix.os != 'sequoia-arm64' }}` based on https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#example-using-contexts, I also realised I need an AND not an OR